toolkits/springernature/packages/springer-nature-context/scss/30-mixins/typography.scss

```diff
@@ -0,0 +1,68 @@
+@mixin u-font-smoothing() {
+	-webkit-font-smoothing: antialiased;
+	-moz-osx-font-smoothing: grayscale;
+}
```

Comment:
shouldn't need this as we get something very similar from `default`
https://github.com/springernature/frontend-toolkits/blob/master/context/brand-context/default/scss/30-mixins/typography.scss#L13